README.rst

```diff
@@ -51,72 +55,123 @@ Build and tests status
 |Develop| |devel-cov|  | |devel-posix|   | |devel-win|  |
 +-------+--------------+-----------------+--------------+
 
+Documentation Build
```

Comment:
How is this different from the documentation link below?

Reply:
This one is the Documentation Build (RTD) status, like the code build status often displayed on README's. The ones below are just links. Check it out [here](https://github.com/AyanSinhaMahapatra/scancode-toolkit/tree/add-doc-link-to-readme)